x/ecocredit/init_test.go

```diff
@@ -0,0 +1,9 @@
+package ecocredit
+
+import (
+	"testing"
+)
+
+func TestSuites(t *testing.T) {
+	// suite.Run(t, &ModelSuite{})
```

Comment:
??

Reply:
Yes, this is from the test I removed. I will remove this file.